inc/resource.class.php

```diff
@@ -551,6 +569,11 @@ function defineTabs($options = []) {
       $this->addStandardTab('PluginResourcesEmployee', $ong, $options);
       $this->addStandardTab('PluginResourcesChecklist', $ong, $options);
       $this->addStandardTab('PluginResourcesTask', $ong, $options);
+      $config = new PluginResourcesConfig();
+      $config->getFromDB(1);
+      if($config->getField('import_external_datas')==1){
```

Comment:
The indentation is not good.
You can only use:  
`if ($config->useImportExternalDatas()) {
`
A method was used to check this parameter why not use it?